src/plot_api/plot_schema.js

```diff
@@ -231,7 +231,12 @@ function getModule(arg) {
             { attributes: polarAreaAttrs } :
             Plotly.Plots.getModule({type: arg.type});
```

Comment:
We should try to move references to Plotly.Plots to just Plots when we come across it. `plots/plots` can stand on its own now I believe.